java/all/src/main/scala/org/typelevel/otel4s/java/OtelJava.scala

```diff
@@ -46,11 +47,19 @@ object OtelJava {
     * @return
     *   An effect of an [[org.typelevel.otel4s.Otel4s]] resource.
     */
-  def forSync[F[_]: LiftIO: Sync](jOtel: JOpenTelemetry): F[Otel4s[F]] = {
+  def forSync[F[_]: LiftIO: Sync](
+      jOtel: JOpenTelemetry
+  ): F[Otel4s[F]] = {
     for {
       metrics <- Sync[F].pure(Metrics.forSync(jOtel))
       traces <- Traces.ioLocal(jOtel)
     } yield new Otel4s[F] {
+      def propagators: ContextPropagators[F] =
```

Comment:
The Otel spec says a lot about what this looks like, but much less about how it's configured.  The ContextConversions are the two things we introduce, and should probably be configurable beyond the SDK's standard mechanisms.